catalyst-gateway-crates/c509-certificate/src/c509_big_uint.rs

```diff
@@ -0,0 +1,87 @@
+//! C509 Unwrapped CBOR Unsigned Bignum
+
+// cspell: words Bignum bignum
+
+use minicbor::{encode::Write, Decode, Decoder, Encode, Encoder};
+
+/// A struct representing an unwrapped CBOR unsigned bignum.
+#[derive(Debug, Clone, PartialEq)]
+pub struct UnwrappedBigUint(u64);
```

Comment:
Should also put the detail about why it's called "unwrapped" here in the docs comment.

Reply:
Will point to a document about CDDL unwrap.